index.html

```diff
@@ -0,0 +1,125 @@
+        <li class="campaign"><a class="nav-item" href="#">GW Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section class="headline">
+      <p class="welcome">"Hi! Sharing world"</p>
+      <h1>10th GLOBAL <br>WAQF CONFERENCE <br>2022</h1>
+      <p class="intro">
+        GWC is the premier Waqf forum that create awareness, develops new and
+        innovative ideas, and ensures the principles of good Waqf governance are widely disseminated and put into
+        practice by people, businesses and leaders as a whole.
+      </p>
+      <h2>2022.12.02(FRI) ~ 04(SUN)</h2>
+      <p class="venu">@ Islamia University, Bahawalpur, Pakistan</p>
+    </section>
+
+    <section id="main-program" class="main-program">
+      <h2>Main Program</h2>
+      <hr>
+      <ul class="program-info">
+        <li>
+          <img src="./Images/Icons/exhibition-icon.png" alt="exhibition-icon">
+          <h3>Inovation</h3>
+          <p>Learn about innovative and new technologies available for waqf developement</p>
+        </li>
+        <li>
+          <img src="./Images/Icons/lecture-icon.png" alt="lecture-icon">
+          <h3>Leadership</h3>
+          <p>Learn from leading global subject matter experts</p>
+        </li>
+        <li>
+          <img src="./Images/Icons/workshop-icon.png" alt="workshop-icon">
+          <h3>Network</h3>
+          <p>Network with international experts, practitioners & academicians</p>
+        </li>
+        <li>
+          <img src="./Images/Icons/forum-icon.png" alt="forum-icon">
+          <h3>Success Stories</h3>
+          <p>Learn about waqf breakthroughs & successful project impementations</p>
+        </li>
+        <li>
+          <img src="./Images/Icons/ignite-icon.png" alt="ignite-icon">
+          <h3>Breakout Sessions</h3>
+          <p>Participate in interactive breakout sessions featuring a wide range of topics</p>
+        </li>
+      </ul>
+      <button id="join-btn">Join Global Waqf Conference 2022</button>
+      <a class="whole-program-btn" href="#">See The Whole Program</a>
+    </section>
+
+    <section id="speakers" class="featured-speakers">
+      <h2>Featured Speakers</h2>
+      <hr>
+      <ul class="speakers-info">
+      </ul>
+      <button class="more-speakers">MORE<i class="fa-solid fa-chevron-down"></i></button>
+    </section>
+
+    <section id="sponsor">
+      <h2>Partner</h2>
+      <hr>
+      <ul class="partner-logo">
+        <li><img src="./Images/partners/Finterra-logo.png" alt="Finterra-logo"></li>
+        <li><img src="./Images/partners/fsmmu-logo.png" alt="fsmmu-logo"></li>
+        <li><img src="./Images/partners/iub-logo.png" alt="iub-logo"></li>
+        <li><img src="./Images/partners/isdb-logo.png" alt="isdb-logo"></li>
+        <li><img src="./Images/partners/mmu-logo.png" alt="mmu-logo"></li>
+      </ul>
+    </section>
+  </main>
+
+  <footer>
+    <h2>© 2022 Global Waqf Conference. Some Rights Reserved.</h2>
+  </footer>
```

Comment:
- [ ] Kindly add a logo beside the text in your `footer` section as seen in the original design 🙏 

Original Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205350939-3bc307cd-2ac3-402f-a5a6-0490c332ecc3.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205350810-98e8e229-7ef3-4567-accc-5045d63a2762.png)

Reply:
[Add an image for footer logo](https://github.com/MNisarAli/1st-Capstone-Conference-page/pull/1/commits/1d0cb70cc1efac5116b5c117535c49d3bae4066c) as per requirement.